Tasks/AzureIoTEdge/task.json

```diff
@@ -0,0 +1,244 @@
+{
+  "id": "80F3F6A0-82A6-4A22-BA7A-E5B8C541B9B8",
+  "name": "AzureIoTEdge",
+  "friendlyName": "Azure IoT Edge",
+  "description": "Azure IoT Edge",
+  "helpMarkDown": "",
```

Comment:
Maybe I need to add links to [docs](https://docs.microsoft.com/en-us/azure/iot-edge/how-to-ci-cd) ? Can I add link to our own [github](https://github.com/Microsoft/azure-iot-edge-vsts-extension) so that we can manage the issues, change log more conveniently? Or is there other best practices?

Reply:
recommend add link to the docs. We need to close on the removal of the second git hub which we can discuss seperately.

Reply:
recommend using an aka.ms url so that the link can be updated offline to point to the right url